crates/sources/src/subgraph.rs

```diff
@@ -4,9 +4,11 @@ use anyhow::{bail, Result};
 use lazy_static::lazy_static;
 use reqwest::{Client, IntoUrl, Url};
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
-use serde_json::{Map, Value};
+use serde_json::{json, Map, Value};
 use thiserror::Error;
 
+const QUERY_PAGE_SIZE: usize = 500;
```

Comment:
Is this the max page size?
Do all paginated queries for all the different types support the same max page size?

Reply:
Balancer uses 1000 page size while Uniswap V3 was tested against 500. I read somewhere that 1000 is max supported by GraphQL but can't find this info now.